packages/hdwallet-keepkey-webusb/src/adapter.ts

```diff
@@ -32,6 +32,8 @@ export class WebUSBKeepKeyAdapter {
   }
 
   private async handleConnectWebUSBKeepKey (e: USBConnectionEvent): Promise<void> {
+    if (e.device.vendorId !== VENDOR_ID) { return }
```

Comment:
mind checking productId too? for webusb KeepKeys, it should be `0x0002`. likewise in the disconnect handler.